setup.py

```diff
@@ -23,23 +23,20 @@
 with open(os.path.join(os.path.dirname(__file__), 'README.md'), 'r') as f:
     long_description = f.read()
 
-
-
-
 setup(
-    name='afs',
+    name='afs2-model',
     version=version,
     description='For AFS developer to develop analytics',
     long_description=long_description,
     author='benchuang',
-    author_email='benchuang@iii.org.tw',
+    author_email='ben2019.chuang@advantech.com.tw',
     url='https://github.com/benchuang11046/afs',
     license='MIT',
```

Comment:
Make sure it is good to use MIT license here. This should be confirmed with your supervisors.

Reply:
If you can't get the answer from your supervisors, I think it is fine to remove this line.

Reply:
The supervisor said that it is better with Apache license.

Reply:
Great, I think this might also need to create a `LICENSE` file under the project root.